server/routes/github/github_app.py

```
@@ -0,0 +1,445 @@
+            frontend_url=FRONTEND_URL,
+        )
+    )
+
+
+@github_app_bp.route("/app/installations", methods=["GET", "OPTIONS"])
+@require_permission("connectors", "read")
+def github_app_list_installations(user_id):
+    """List GitHub App installations linked to the requesting user."""
+    try:
+        with db_pool.get_admin_connection() as conn:
+            with conn.cursor() as cur:
```

Ruff (0.15.12)

[warning] Use a single `with` statement with multiple contexts instead of nested `with` statements

Combine `with` statements

(SIM117)